cypress/Objects/loginPage.js

```diff
@@ -0,0 +1,25 @@
+class LoginPage {
+    visit(){
+        cy.visit('/');
+    }
+    getvalidationerror() {
+        return cy.get('.Select.not-valid');
+    }
+    submitlogin(){
```

Comment:
ccording to camel case description (a convention for file/variable naming) every new word in variable name should start with capital letter, so it should be submit**L**ogin